cmd/kubelet/app/server.go

```diff
@@ -94,8 +94,9 @@ containers which were not created by Kubernetes.
 Other than from an PodSpec from the apiserver, there are three ways that a container
 manifest can be provided to the Kubelet.
 
-File: Path passed as a flag on the command line. This file is rechecked every 20
-seconds (configurable with a flag).
+File: Path passed as a flag on the command line. Path and files under this path will
+always be watched, and this work will be restarted after 20 seconds(configurable with
```

Comment:
yes, this expression is more clearly, thank you for your suggest, i have changed it!